docker.readme

```diff
@@ -0,0 +1,35 @@
+## The following command should be runned after the yocto
+## project has been setup (after calling setup-angstrom-manifest.sh).
+## If building for the first time you will need to run all 3
+## commands.
+
+# Build docker image that we can use to create a container
+# This should be runned from angstrom-v2019.12-zeus-fluke-cda-manifest
+# Run once only
+docker build -t fluke/nighthawk:latest \
+    --build-arg USER=$USER \
+    --build-arg USER_ID=$(id -u) \
+    --build-arg GROUP_ID=$(id -g) \
+    --build-arg PWD=$PWD \
+    .
+
+# Verify your image has been built
+docker image ls
+
+# Create and run our container
+# Run once
+docker run \
+    -d \
+    --mount type=bind,source=$PWD,target=$PWD \
+    --mount type=bind,source=$HOME/.ssh,target=$HOME/.ssh \
+    --hostname nh-yocto \
+    --name nh-yocto \
+    fluke/nighthawk
+
+
+# Verify your container has been created and is running
+docker container ls
+
+# Open a bash shell in our container
+# Run as needed
+docker exec -it nh-yocto bash
```

Comment:
You can also have  'docker run --rm ...' which will remove the container when done.  Since the output of the docker container is being save to the host OS system, this is a valid alternative way.   At least add a commend above the --rm flag to 'docker run' as it is easy to fill the local file system for the unaware.  You might also add info above 'docker image ls' vs 'docker container ls' and how to remove unneeded resources.